app/routes/individual_response.py

```diff
@@ -0,0 +1,320 @@
+
+    language_code = get_session_store().session_data.language_code
+    session_timeout = get_session_timeout_in_seconds(schema)
+
+    if request.method == "GET":
+        individual_id = request.args.get("individual_id")
+        previous_location_url = (
+            f"/questionnaire/household/{individual_id}/individual-interstitial/"
+        )
+
+        return render_template(
+            template="individual-response",
+            session_timeout=session_timeout,
+            language=language_code,
+            previous_location_url=previous_location_url,
+        )
+
+    individual_id = request.args.get("individual_id")
+
+    return redirect(
+        url_for(".get_individual_response_how", individual_id=individual_id)
+    )
+
+
+@individual_response_blueprint.route("/<individual_id>/how", methods=["GET", "POST"])
+@login_required
+@with_questionnaire_store
+@with_schema
+def get_individual_response_how(schema, questionnaire_store, individual_id):
+    router = Router(
+        schema,
+        questionnaire_store.answer_store,
+        questionnaire_store.list_store,
+        questionnaire_store.progress_store,
+        questionnaire_store.metadata,
+    )
+
+    if not router.can_access_hub():
+        return redirect("/questionnaire")
+
+    language_code = get_session_store().session_data.language_code
+    session_timeout = get_session_timeout_in_seconds(schema)
+    previous_location_url = f"/individual-response?individual_id={individual_id}"
+
+    first_name = questionnaire_store.answer_store.get_answer(
+        "first-name", individual_id
+    ).value
+    last_name = questionnaire_store.answer_store.get_answer(
+        "last-name", individual_id
+    ).value
+    name = f"{first_name} {last_name}"
+
+    block_definition = {
+        "type": "Question",
+        "id": "individual-response-how",
+        "title": lazy_gettext(
+            "How would you like <em>{name}</em> to receive a separate census?".format(
+                name=name
+            )
+        ),
+        "description": lazy_gettext(
+            "<p>For someone to complete a separate census, we need to send them an individual access code.</p><p>Select how to send access code</p>"
+        ),
+        "answers": [
+            {
+                "type": "Radio",
+                "id": "individual-response-how-answer",
+                "mandatory": True,
+                "options": [
+                    {
+                        "label": lazy_gettext("Post"),
+                        "value": "Post",
+                        "description": lazy_gettext(
+                            "We can only send this to an unnamed resident at the registered household address"
+                        ),
+                    }
+                ],
+            }
+        ],
+    }
+
+    form = generate_form(
+        schema,
+        block_definition,
+        None,
+        questionnaire_store.metadata,
+        form_data=request.form,
+    )
```

Comment:
Would be nice for this to be named args, as I always have to look into the method when passing like None/empty data structure.